src/cmake-tools.ts

```diff
@@ -260,9 +260,14 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
         if (result === quickStart) {
           vscode.commands.executeCommand('cmake.quickStart');
         } else if (result === changeSetting) {
+          // Open the search file dialog from the path set by cmake.sourceDirectory or from the current workspace folder
```

Comment:
I thought the focus of this flow was that `cmake.sourceDirectory` is not set and we are going to set it now.  I don't think we should try to create the driver at this location in the code.  Just use the workspace folder for now.

Reply:
As discussed offline, I reverted both changes regarding using sourceDir versus workspace folder (which require driver instantiation) to not risk any destabilization for the 1.4 release and I will open a new issue (low priority) describing the scenarios that are impacted by this to review later.